src/clj/rems/api/extra_pages.clj

```diff
@@ -0,0 +1,37 @@
+(ns rems.api.extra-pages
+  (:require [clojure.java.io :as io]
+            [compojure.api.sweet :refer :all]
+            [rems.api.schema :refer :all]
+            [rems.config :refer [env]]
+            [rems.common-util :refer [index-by]]
+            [ring.util.http-response :refer :all]
+            [schema.core :as s])
+  (:import (java.io FileNotFoundException)))
+
+(s/defschema ExtraPage
+  {s/Keyword s/Str})
```

Comment:
this is shadowing the ExtraPage schema from rems.api.schema, please remove it

Reply:
Done